app/models/services/facebook.rb

```diff
@@ -23,16 +23,16 @@ def post_to_facebook(url, body)
   end
 
   def create_post_params(post)
-    message = post.message.plain_text_without_markdown
-    if post.photos.any?
-      message += " " + short_post_url(post, protocol: AppConfig.pod_uri.scheme,
-                                            host: AppConfig.pod_uri.authority)
+    post_message = post.message.plain_text_without_markdown
```

Comment:
Why did you rename this to `post_message` but then use again `message` below? I think one variable should be enough.

Reply:
Code readability since that is explicitly the post message versus the message that is being embedded into the return object.  I can use one variable throughout if that is preferred.